main/SAPI.c

```diff
@@ -458,6 +458,10 @@ SAPI_API void sapi_activate(void)
 	SG(request_parse_body_context).throw_exceptions = false;
 	memset(&SG(request_parse_body_context).options_cache, 0, sizeof(SG(request_parse_body_context).options_cache));
 
+	if (sapi_module.pre_request_init) {
+		sapi_module.pre_request_init();
```

Comment:
Return value not checked?

Reply:
This is same as activate - also not checked and always return SUCCESS. I think it still makes sense to keep the return codes in case we decided to change the meaning and somehow fail but it's not necessary atm.